src/components/Navbar.tsx

```diff
@@ -1,4 +1,14 @@
+import { NavLink, useSearchParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { ReactNode } from 'react';
+
+const activeLinkClasses = ({ isActive }: { isActive: ReactNode }) => {
```

Comment:
The `isActive` prop is typed as `ReactNode`, which is not correct. The `isActive` prop should be a boolean, indicating whether the link is active or not. Please, correct the type of the `isActive` prop.